tests/fixtures/given.go

```diff
@@ -110,29 +113,49 @@ func (g *Given) readResource(text string, v metav1.Object) {
 
 // initialize git repo with all Numaflow files
 func (g *Given) CloneGitRepo() *Given {
```

Comment:
maybe we should rename this function to something like `InitializeGitRepo()` since it does more than just clone, it actually pushes files too

Reply:
also, if in particular it's initializing with particular files, a comment would be helpful to indicate that